src/Microsoft.AspNetCore.Mvc.DataAnnotations/Internal/DataAnnotationsMetadataProvider.cs

```diff
@@ -20,6 +21,19 @@ public class DataAnnotationsMetadataProvider :
         IDisplayMetadataProvider,
         IValidationMetadataProvider
     {
+        private IStringLocalizerFactory _stringLocalizerFactory { get; set; }
+
+        public DataAnnotationsMetadataProvider()
+        {
+
+        }
+
+        public DataAnnotationsMetadataProvider(IStringLocalizerFactory stringLocalizerFactory)
+            : base()
```

Comment:
Also, the other constructor is empty (ignoring the momentous blank line). Why call it at all?